illustrations-backgrounds.html

```diff
@@ -0,0 +1,2277 @@
+---
```

Comment:
**** poti pune aici datele din _data background si sa le folosesti in _head.html cu page.title, page.canonical in loc de site.data.canonical. Astefel reunti la if-ul imens in head.html si la yml-urile din data